code/script.js

```diff
@@ -1,9 +1,12 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
 function toggle() {
   this.classList.toggle("active")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("nav").onclick = toggle
```

Comment:
smooth, your js-code made taught me how to not have like 50 lines of unnecessary code. Smooth! Also, smart to use the same function for the nav!